### ISSUE TITLE
Tensorflow implementation not compiling?

### ISSUE THREAD

**Issue by maksay** [teganmaharaj/zoneout#2]
As I missing something, or both **zoneout_tensorflow** and **zoneout_seq2seq** feature non-existing variables and wrong function parameter names?

**Comment by ASalvail** [comment]
Indeed, `state_part_zoneout_prob` is referenced in call but isn't initialized anywhere in the float version.
However, if you happen to have your zoneout probabilities in a numpy array and the `state_size` is a tuple, the computation will actually go through without problems. The code should still be fixed, but at least there is a working path.

**Comment by ASalvail** [comment]
Be careful however since doing this is actually converting your tuple into a 2-dim numpy array. It works, but honnestly, it shouldn't.
